library/common.py

```diff
@@ -70,3 +71,30 @@ def generate_text(seed_text, next_words, model, max_sequence_len):
                 break
         seed_text += " "+output_word
     return seed_text.title()
+
+
+def _random_prefix(sentences):
+    """
+    prefix random generator
+    input: list of input sentences
+    output: random word
+    """
+
+    words = _word_dict(sentences)
+    return choice(words)
+
+def _word_dict(sentences):
+    """
+    input: list of input sentences
+    output: unique list of the corpus
```

Comment:
`unique list of the corpus` --> `unique list of words in the corpus` ? Does a `corpus` imply uniqueness?